rdm/0002-data-models-terminology.md

```diff
@@ -0,0 +1,38 @@
+- Start Date: 2019-09-30
+- RFC PR: [#2](https://github.com/inveniosoftware/rfcs/pull/2)
+- Authors: Lars Holm Nielsen
+
+# Data models terminology
+
+## Summary
+
+Definition of terms used when speaking about records and data models.
+
+## Motivation
+
+The purpose of this RFC is to disambiguate terms like records, record type, schema and data model that are often used interchangeably in the context of Invenio.
+
+## Detailed design
+
+The central part of InvenioRDM is the primary data model for storing a bibliographic record type describing the digital resources (datasets, software, articles, ...). In addition to the primary data model, InvenioRDM can be extended with one or more secondary data models for storing authority record types (grants, licenses, ...).
+
+You may often see the words like records, record type, schema and data model used interchangeably in the context of Invenio, therefore it’s useful to distinguish what is meant with each term:
+
+- **Record:** a JSON document describing an object.
+- **Record type:** a category of JSON documents describing a specific type of objects (examples: grants, licenses, authors, research outputs).
```

Comment:
Actually I think it would help. But we should add some clear examples. I was wondering the same thing...isn't record type 'Software'? but of course, it is not, it is a resource type.

Reply:
Maybe we could add something like "Do not confuse with record's resource type, e.g. paper, poster, software, etc. All of them belong to the same record type 'bibliographic record'...Or something similar.